portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,19 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random photo.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function randomizeImage() {
+  // The images directory contains 17 images, so generate a random index between
+  // 1 and 17.
+  const imageIndex = Math.floor(Math.random() * 17) + 1;
```

Comment:
Hardcoding a number "17" to the code means that we need to update the number whenever we add/remove images in the folders. Two possible ways to improve it:
1) Put all images into a single folder and read the folder to determine the number of images in the folder. Also, you don't need to fix the prefix to "lagos-". You might be able to get the file name directly.
2) If the option 1 is too difficult, you can put a file close to the folder of images, and add the hardcode value there. Or you can extract the hardcode value to the top of this file. In this way, people don't need to scroll down the file to find the hardcode value when they need to modify it in the future.

Reply:
i extracted the hardcoded value to the top as a variable so i can be easily modified. Thanks!